### ISSUE TITLE
Revoir le formulaire contact Wordpress

### ISSUE THREAD

**Issue by Mathieu-CdlT** [chemins-de-la-transition/plateforme#54]
Reprendre et simplifier le formulaire contact. 1 seule case d'écriture quitte à suggérer aux répondants quoi mettre dedans (besoins, compétences, envies...). 

Voir pourquoi les mails reçus issus de ces formulaires contacts sont vides souvent (les répondants se contentent de remplir les cases?). 

Toutes les cases sont utiles ?

**Comment by GuillaumeAV** [comment]
Oui, il faut surtout changer de plugin ... Mais en effet, une seule case pourrait suffire .. 
J'essaie de regarder la semaine prochaine !
